tasty/src/dotty/tools/tasty/TastyFormat.scala

```diff
@@ -318,7 +318,7 @@ object TastyFormat {
    *  compatibility, but remains backwards compatible, with all
    *  preceeding `MinorVersion`.
    */
-  final val MinorVersion: Int = 4
+  final val MinorVersion: Int = 5
```

Comment:
```suggestion
  final val MinorVersion: Int = 4
```

See  #13447

Reply:
I'm even more confused now. We are now setting up a tasty version for the nightly releases. According to [this diagram](https://github.com/lampepfl/dotty/issues/13447#issuecomment-912447107) and [this table](https://github.com/lampepfl/dotty/issues/14306#issuecomment-1069333516). When we have a nightly version in form 3.x.y, where y > 0, tasty should have version 28.(x+1)-exp1. That means it should be 28.5.1 now, as we will release nightlies for 3.4.1.

Reply:
Right, based on those diagrams the Nighlies should be on 28.5. I do not know why nightlies should have a different version. It seems it would make more sense for them to have the same version. I guess that the current check does not take this into account. @bishabosha could you clarify this?

Reply:
We should be able to update the TASTy minor version using 3.3.1 for now. See #19483. We can later update the reference version to `3.4.0-RC2`. We just need to make sure we have the correct version for that one.

Reply:
> Right, based on those diagrams the Nighlies should be on 28.5. I do not know why nightlies should have a different version. It seems it would make more sense for them to have the same version. I guess that the current check does not take this into account. @bishabosha could you clarify this?

Nightlies should be 1 minor version ahead because experimental is treated as a preview - so `28.4-exp-1` can only read `3.3.x` compiled code

Reply:
> however the 3.4.1-RC1 final must also have 28.4-0 tasty version

Should it be  `28.4-0` or ` 28.4-2` as in [the diagram](https://github.com/lampepfl/dotty/issues/13447#issuecomment-912447107)?

Reply:
the `-2` is only for RC when patch version is 0